bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/client/model/ApiRequest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.homeconnect.internal.client.model;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ *
+ * API request model.
+ *
+ * @author Jonas Brüstel - Initial Contribution
+ */
+@NonNullByDefault
+public class ApiRequest {
+    private final String id;
+    private final ZonedDateTime time;
+    private final HomeConnectRequest homeConnectRequest;
+    private final @Nullable HomeConnectResponse homeConnectResponse;
+
+    public ApiRequest(ZonedDateTime time, HomeConnectRequest homeConnectRequest,
```

Comment:
Is the timezone information useful here?
Should be `java.time.Instant` in my opinion.
(Same  goes for all usages of `ZonedDateTime`

Reply:
Yes it is very useful to debug problems. I just changed it before creating this pull request. One tester was in a different time zone and it was very confusing.

Reply:
Where was the `Instant` logged?
It should be serialized as ISO8601 string which is unambiguous.

Reply:
The `Instant` is visible in the binding UI request and event log.
![Home Connect Console](https://user-images.githubusercontent.com/2088078/102503796-60dcb280-4080-11eb-9c99-e5a016c3674c.png)

I also use it in the event and request log export (json format).
![Bildschirmfoto von 2020-12-17 15-54-32](https://user-images.githubusercontent.com/2088078/102504105-b44f0080-4080-11eb-90b1-8ee22ef687ec.png)

I'm not persisting the `Instant` object (like a log file etc.), it's only used in a FIFO queue.